### ISSUE TITLE
Add support for different borderRadius values in the 4 corners

### ISSUE THREAD

**Issue by scailbc** [lightning-js/renderer#132]
Hello, is it possible to improve the `borderRadius` property to define different values for the four corners? The same as the CSS property if possible.
If that is too much work, we can simply add extra properties like `border-top-left-radius`, that should be quick amd easy to do.
Thanks

**Comment by chiefcll** [comment]
Are you using one of the frameworks?

borderRadius takes an array of values [4, 8, 8, 4] for the four corners

**Comment by scailbc** [comment]
Yes, I'm using lightning/solid and the array works. The interface `IntrinsicNodeStyleCommonProps` and the docs should be updated, because it only says number and from the source code you can't tell that an array is acceptable.
Can we do that?

**Comment by chiefcll** [comment]
Fixed - update solid to 0.11.4
